Lib/test/test_sys.py

```diff
@@ -1032,6 +1032,10 @@ def test_implementation(self):
         self.assertEqual(sys.implementation.name,
                          sys.implementation.name.lower())
 
+        if hasattr(sys.implementation, '_architecture'):
```

Comment:
This should check if the system is Windows, not if `sys.implementation` has it. If someone were to accidentally remove it in the future, this test would still pass.

Reply:
Maybe I can add a condition.